src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -165,27 +165,27 @@
     },
     "formattedMessage": 
     {
-      "description": "A formatted message object encapsulates information that can be used to construct a fully formatted message that describes an issue.",
+      "description": "Contains information that can be used to construct a formatted message that describes a result.",
       "additionalProperties": false,
       "type": "object",
       "properties": {
-        "specifierId": 
+        "formatId": 
         {
-          "description": "A string that identifies the format string used to format the message that describes this result. The value of specifierId must correspond to one of the names in the set of name/value pairs contained in the format specifiers property of the rule info object whose id property matches the rule id property of this issue.",
+          "description": "A string that identifies the message format used to format the message that describes this result. The value of formatId must correspond to one of the names in the set of name/value pairs contained in the 'messageFormats' property of the rule object whose 'id' property matches the 'ruleId' property of this result.",
           "type": "string"
         },
 
         "arguments": 
         {
-          "description": "An array of string values that will be used, in combination with a format specifier, to construct a result message.",
+          "description": "An array of strings that will be used, in combination with a message format, to construct a result message.",
           "type": "array",
           "items":
           {
             "type": "string"
           }
         }
       },
-      "required": ["specifierId", "arguments"]
+      "required": ["formatId"]
```

Comment:
ok, so users can pass no arguments? retrieving a static string? hm. i guess this makes sense.